app/charts/map/map.tsx

```diff
@@ -170,6 +171,7 @@ export const MapComponent = () => {
           });
         }
       },
+      // @ts-ignore
```

Comment:
It'd be good to add the reason why we ignore here.

Reply:
It's due to typed version of Deckl.gl which started to throw type errors here, while it was (and still is) working properly. I tried to fix them, but wasn't able to do it in reasonable amount of time – I'll add a todo here to remember to come back to it.